### PR TITLE
Adding Hungarian translation to nightscout

### PR DESCRIPTION
**Pull request by andrasfeher-smilodonis**
The PR is to add Hungarian language to nightscout. All required variables have been translated.

The tranlations have been tested and is working properly. Some of the texts are not getting translated but that is a general issue and I opened a ticket for that.

Also would like to contribute to nightscout in Hungarian and Slovak language as I speak both very well. Please let me know if there is an other way to ask to become a contributor as this is my first contribution to Nightscout. 

Thank you.

### CODE REVIEW

**Comment by andrasfeher-smilodonis** [comment]
@sulkaharo Do I need to do some additional work to get this PM reviewed? Thank you!

**Comment by sulkaharo** [comment]
Hi! No, sorry - I was just super busy with the mission critical issues. Merging soon!

**Comment by andrasfeher-smilodonis** [comment]
@sulkaharo **Thank you.** No hurry, I know there are more important stuff to do and fix. I just wanted to make sure I did all I needed to do. Also when you do the merge, please let me know how to become a regular contributor the languages mentioned above in the PR text other than to actually create PRs.

File: lib/language.js [hunk 1]
```diff
@@ -14330,7 +14328,6 @@ function init() {
       , tr: 'CGM Transmitter Age'
       , zh_cn: 'CGM Transmitter Age'
       , zh_tw: 'CGM Transmitter Age'
-      , zh_tw: 'CGM jeladó életkora'
```
Comment (andrasfeher-smilodonis) [inline lib/language.js]: The yh_tw should be changed hu

File: lib/language.js [hunk 2]
```diff
@@ -15229,7 +15226,6 @@ function init() {
       , hr: 'Unknown Intent'
       , pl: 'Unknown Intent'
       , ru: 'Неизвестное намерение'
-      , tr: 'Unknown Intent'
       , tr: 'Ismeretlen szándék'
```
Comment (andrasfeher-smilodonis) [inline lib/language.js:15229]: Second tr should be changed to "hu"

**Comment by sulkaharo** [comment]
Hey I ran this locally and looks like it'd be great if you could do one more pass on checking what all the text looks like in the actual user interface. Some of the text is so long the user interface layouts look pretty strange (yes I know this is unavoidable in many cases) and some keys look odd for other reasons, like the Clock navigation.

**Comment by andrasfeher-smilodonis** [comment]
Hi, thank you. I added some comments to two lines in language.js to fix. I will recheck all the texts if there can be some additional fixes. I'll let you know tonight.

File: lib/language.js (hunk 2 above)
Comment (andrasfeher-smilodonis) [inline lib/language.js:15229]: Second tr should be changed to "hu"

**Review by andrasfeher-smilodonis — COMMENTED** [review]
Some fixed added

File: lib/language.js (hunk 1 above)
Comment (andrasfeher-smilodonis) [inline lib/language.js]: Change second "zh_tw" to "hu"

**Review by andrasfeher-smilodonis — COMMENTED** [review]
Fix

**Comment by andrasfeher-smilodonis** [comment]
@sulkaharo Hi. As requested, I went through the site locally but not really sure what you mean by "Clock navigation." I see some places where the text might be a bit longer but nothing major in my opinion. The translation looks OK. Also did some comments on your edits, see above. I find some issues with translation in general, (I opened a ticket fir it #5991) - there should be a fix for it in #6217. 

If you have the time please review this and approve or let me know what to add/edit/change. 

Thank you , Andy.

**Pull request merged**